client/my-sites/earn/home.tsx

```diff
@@ -0,0 +1,196 @@
+				{
+					components: {
+						em: <em />,
+					},
+				}
+			),
+			image: {
+				path: '/calypso/images/earn/recurring.svg',
+			},
+			cta: {
+				feature: FEATURE_NO_ADS,
+				upgradeButton: {
+					text: translate( 'Upgrade to a Paid Plan' ),
+					action: () => page( `/plans/${ selectedSiteSlug }` ),
+				},
+				defaultButton: {
+					text: translate( 'Collect Recurring Payments' ),
+					action: () => page( `/earn/payments/${ selectedSiteSlug }` ),
+				},
+			},
+			learnMoreLink: 'https://en.support.wordpress.com/recurring-payments/',
+		};
+	};
+
+	/**
+	 * Return the content to display in the Store card based on the current plan.
+	 *
+	 * @returns {object} Object with props to render a PromoCard.
+	 */
+	const getStoreCard = () => {
+		// Todo: get plan and return different content based on it.
+		return {
+			title: translate( 'Sell a few items' ),
+			body: translate(
+				'Create a basic store with minimal setup -- add your products, and use the built-in shipping, coupon, and tax tools. {{em}}Available to sites with a Business plan{{/em}}.',
+				{
+					components: {
+						em: <em />,
+					},
+				}
+			),
+			image: {
+				path: '/calypso/images/earn/woo.svg',
+			},
+			cta: {
+				feature: FEATURE_UPLOAD_PLUGINS,
+				upgradeButton: {
+					text: translate( 'Upgrade to Business Plan' ),
+					action: () => page( `/checkout/${ selectedSiteSlug }/business/` ),
+				},
+				defaultButton: {
+					text: translate( 'Set Up a Simple Store' ),
+					action: () => page( `/store/${ selectedSiteSlug }` ),
+				},
+			},
+			learnMoreLink: 'https://en.support.wordpress.com/store/',
+		};
+	};
+
+	/**
+	 * Return the content to display in the Ads card based on the current plan.
+	 *
+	 * @returns {object} Object with props to render a PromoCard.
+	 */
+	const getAdsCard = () => {
+		// Todo: get plan and return different content based on it.
+		return {
+			title: translate( 'Earn ad revenue' ),
+			body: translate(
+				'Publish as you normally would, display advertisements on all your posts and pages, and make money each time someone visits your site. {{em}}Available to sites with a Premium plan{{/em}}.',
+				{
+					components: {
+						em: <em />,
+					},
+				}
+			),
+			image: {
+				path: '/calypso/images/earn/ads.svg',
+			},
+			cta: {
+				feature: FEATURE_WORDADS_INSTANT,
+				upgradeButton: {
+					text: translate( 'Upgrade to Premium Plan' ),
+					action: () => page( `/checkout/${ selectedSiteSlug }/premium/` ),
+				},
+				defaultButton: {
+					text: translate( 'Earn Ad Revenue' ),
+					action: () => page( `/earn/ads-earnings/${ selectedSiteSlug }` ),
+				},
+			},
+			learnMoreLink: 'https://wordads.co/',
+		};
+	};
+
+	const promos: PromoSectionProps = {
+		header: {
+			title: translate( 'Start earning money' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **22** times:
`translate( 'Start earning money now' )` *ES Score: 12*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

Reply:
@davemart-in Do you think we should add 'now' into the title of the header component?

Reply:
That's fine I think. 👍